dbschema/budget-record.esdl

```diff
@@ -0,0 +1,13 @@
+module default {
+  type BudgetRecord extending Project::ContextAware {
+    required fiscalYear: int16;
+    required amount: float32 {
+      default := 0;
+    };
```

Comment:
Let's make this optional. We auto create these, so the presence of a value here is the only way we know if it's been touched or not. And I think 0 is a valid value people enter.

Also matches app code: https://github.com/SeedCompany/cord-api-v3/blob/e9007a81ac14a3a6269e42fa0f4d2c8c6dc87529/src/components/budget/dto/budget-record.dto.ts#L38-L38
```suggestion
    amount: float32;
```